lib/kitchen/verifier/inspec.rb

```diff
@@ -198,6 +199,13 @@ def runner_options(transport, state = {}, platform = nil, suite = nil) # rubocop
           runner_options["output"] = config[:output] % { platform: platform, suite: suite } unless config[:output].nil?
           runner_options["profiles_path"] = config[:profiles_path] unless config[:profiles_path].nil?
           runner_options[:controls] = config[:controls]
+
+          # check to make sure we have a valid version for caching
+          if config[:backend_cache]
+            backend_cache_msg = "backend_cache requires InSpec version >= 1.47.0"
+            logger.warn backend_cache_msg if Gem::Version.new(::Inspec::VERSION) < Gem::Version.new("1.47.0")
+          end
+          runner_options[:backend_cache] = config[:backend_cache].nil? ? true : config[:backend_cache]
```

Comment:
This places the default behavior of the caching in two places - InSpec, and kitchen-inspec. And technically, this is a breaking change by defaulting it to `true`. We can get away with that since we're still technically `0.x` SemVer version, but I still think it's the wrong choice.

I think it would be better to let this logic live with InSpec until there's a specific use case where we have to deviate the default behavior. Therefore, I'd change this to be:

```ruby
if config[:backend_cache]
  backend_cache_msg = "backend_cache requires InSpec version >= 1.47.0"
  logger.warn backend_cache_msg if Gem::Version.new(::Inspec::VERSION) < Gem::Version.new("1.47.0")

  runner_options[:backend_cache] = config[:backend_cache]
end
```

Reply:
After talking it over with Adam we decided to have it default to false here. This is due to how we are grabbing the internals of inspec without going through the normal default option initiation.